### ISSUE TITLE
branching process stochastic 

### ISSUE THREAD

**Issue by ramyanagarajan06** [RMHogervorst/cleancode#59]
please give me an any example of using branching process for stochastics using r languages.

**Comment by RMHogervorst** [comment]
Thank you for opening a issue! 

I'm sorry I don't understand the question, could you rephrase it?

**Comment by RMHogervorst** [comment]
Hi @ramyanagarajan06, I will close this issue, because I don't understand your question. 
Feel free to open a new issue with the question rephrased.
